presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -283,8 +255,15 @@ private static String buildMapValue(Row row, int position, CassandraType keyType
         return sb.toString();
     }
 
+    private static String buildArrayValue(Row row, int position)
+    {
+        Definition definition = row.getColumnDefinitions().asList().get(position);
```

Comment:
You can replace `.asList().get(position).getType()` with `.getType(position)`

```suggestion
        Type type = row.getColumnDefinitions().getType(position);
```